packages/utilities/src/CliUtils.ts

```diff
@@ -528,9 +530,11 @@ export class CliUtils {
         }
 
         // Ensure that we use a reasonable timeout
-        const maxSecToWait = 900; // 15 minute max
```

Comment:
i thought the 15 minute wait came out of some parking lot discussion on this topic... i forget the details and don't have an email.  i'm just sharing my own recollection

Reply:
I remember we discussed what the default timeout should be (10 min), but I guess I forgot the discussion about the maximum timeout 😢 IMO 15 minutes seems to be kind of an arbitrary limit, but I can change it back if you'd like 🙂